server/src/test/java/org/eclipse/lsp/cobol/usecases/TestCommentLines.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2022 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+
+package org.eclipse.lsp.cobol.usecases;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.eclipse.lsp.cobol.usecases.engine.UseCaseEngine;
+import org.junit.jupiter.api.Test;
+
+/** This test checks multiple comment entries are parsed and cleaned up correctly */
+class TestCommentLines {
+
+  private static final String TEXT =
+      "      * Copyright (c) 2021 Broadcom.\n"
+          + "      * The term Broadcom  refers to Broadcom Inc. and/or its subsidiaries.\n"
+          + "000010 IDENTIFICATION DIVISION.                                         qweasdzx\n"
+          + "000020*    Comment line                                                 qweasdzx\n"
+          + "\n"
+          + "000030 PROGRAM-ID. comments.    *> Floating comment                     qweasdzx";
+
+  @Test
+  void test() {
+    UseCaseEngine.runTest(TEXT, ImmutableList.of(), ImmutableMap.of());
```

Comment:
This is good for debugging but didn't test your changes. Because the test passes even without your changes.
Do the test around `TextPreprocessor::cleanUpCode`. You can find example of usage in `CobolLanguageEngine::run`:
```
String cleanText = preprocessor.cleanUpCode(documentUri, text).unwrap(accumulatedErrors::addAll);
```